### ISSUE TITLE
Decode escaped strings

### ISSUE THREAD

**Issue by christiansr85** [urin/jquery.balloon.js#16]
Hello,

I'm using the last version of your plugin and it's working very fine. But recently I've found an issue related to encoded strings. 

The problem is that sometimes I have to show a text which has double quotes. Storing it in the `title` field of the dom element is a problem because we can't do something like this (please realize about the double quotes inside the title):
```    
<a title="Text with "double quotes" inside">Text with "double quotes" inside</a>
```
So I have encoded the string inside the `title` fields through `encodeURIComponent` function. Perhpaps it could be added the `decodeURIComponent` before the balloon is displayed. By this way we can display the entire strings as they were written originaly even if they have any other troubled chars as `<`, `>`...

I've tried it adding this code in line 206:
```
contents = decodeURIComponent(contents);
```
just before this statement:
```
isNew = !($balloon = $target.data("balloon"));
```

Even if it were interesting to maintain the escaped chars, you could add a flag to the options to indicate if the decode function should be executed or not.

If you have any question or if you want more info about this, please don't hesitate to ask me.

**Comment by urin** [comment]
It is invalid to use double quotes as a value of a DOM. It is restricted as specification of HTML in RFC.
If you want to use double quotes as the title of the element, you should escape double quotes with character reference like `&quot;`.

The `contents` is not a text so that we can place any HTML elements.

**Comment by christiansr85** [comment]
I have tried what you say and it works, of course, but I have to encode the html characters in order to not show them in the tooltip as html code, but as strings. For instance, if a user inserts a comment like this: 'This is a "comment" with <span>html</span>' I would like to show exactly that in the tooltip. So what I do for this is to encode that string and, adding the instruction to the balloon I suggested in my first post, the output of the tooltip displays the original comment.

Besides, the 
```
contents = decodeURIComponent(contents);
```
instruction is transparent to user. I mean, although user wants to pass html code and display it as html in the tooltip, if the text has not been previously encoded, my addition has no effect on the output, so it doesn't modifies anything.

Obviously, it is just an idea:)

And sorry for the delay, but I couldn't answer before.
